src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -418,28 +484,125 @@ def _check_private_endpoint(cmd, registry_name, vnet_of_private_endpoint):  # py
     else:
         raise CLIError('DNS routing verification failed')
 
+# Validate --image input
+# Validate if image layer can be pulled
+def _validate_image_name(repository_name,
```

Comment:
Nit. _validate_image_in_registry is probably more accurate since you are checking for the repository@digest combo.

Reply:
True, I will change this so it's a bit more precise and informative, thank you!